### ISSUE TITLE
Check it’s clear what information users need to provide

### ISSUE THREAD

**Issue by Angelsaur** [GeorgiaHCA/AutSPACEs#52]
### Summary

As you go through your sample forms, you’ll need to check you’re using field labels in a way that makes clear what information the user needs to enter.

### Deadline 

20/02/2021

### What needs to be done?

As you go through your sample forms, you’ll need to check you’re using field labels in a way that makes clear what information the user needs to enter.

As a general rule, these labels should be specific. Look at your forms and make an assessment as to whether they’re clear. Common mistakes include not labelling fields at all, or using vague labels like ‘Name’ which do not make clear whose name the user needs to enter (for example, it could be a partner or child’s name, rather than the user’s own name)
